nautobot/utilities/filters.py

```diff
@@ -389,6 +489,10 @@ def _generate_lookup_expression_filters(cls, filter_name, filter_field):
         field_name = filter_field.field_name
         field = get_model_field(cls._meta.model, field_name)
 
+        # If there isn't a model field, return.
+        if field is None:
+            return magic_filters
```

Comment:
This was required because it was trying to introspect `q` now that it's not a method filter, and since that's not a model field, was failing.

Reply:
Was wondering how you caught this fail condition. 👍